src/index.js

```diff
@@ -33,9 +33,18 @@ refreshButton.addEventListener('click', () => {
 });
 
```

Comment:
- [ ] your README should contain all that's needed for running the app with the API(s) it consumes, kindly adds a `Getting  Started` section for example, and include all the steps to run the app so all the users can get your app and run it easily (you could add steps from cloning the repo to running the app)